plugins/TodayAtMun/diary_parser.py

```diff
@@ -0,0 +1,39 @@
+from bs4 import BeautifulSoup
+import requests
+import os
+from os import path
+import json
+from pathlib import Path
+
+
+class diary_parser:
+    def __init__(self):
+        BASE_DIR = Path(__file__).parent
+        file_name = BASE_DIR / "diary.json"
+        try:
+            Path.unlink(file_name)
+        except FileNotFoundError:
+            Path.touch(file_name)
+        r = requests.get(
+            "https://www.mun.ca/regoff/calendar/sectionNo=GENINFO-0086", verify=False
+        ).text
+        soup = BeautifulSoup(r, "html.parser")
+
+        with open(file_name, "w") as file:
+
+            data = soup.find_all("td", attrs={"align": "left"})
```

Comment:
maybe use the names leftAlignedData and rightAlignedData instead of data and data?

Reply:
Fair, those should've been temporary names. Will Change!